tests/test_backed_sparse.py

```diff
@@ -270,13 +270,41 @@ def test_dataset_append_memory(
 
 
 @pytest.mark.parametrize("sparse_format", [sparse.csr_matrix, sparse.csc_matrix])
+@pytest.mark.parametrize(
+    ("subset_func", "subset_func2"),
+    map(
+        lambda x: pytest.param(*x),
+        product(
```

Comment:
- you don’t need `pytest.param` if you don’t use its keyword arguments, you can just use tuples for that. And `product` already emits tuples
- (no longer relevant if you just use plain `product`, but it’s more pythonic to use comprehensions instead of map with a lambda)
- `product` has a `repeat` parameter that you can use for much less confusing brace syntax.